typings/generated/entities/locale.d.ts

```diff
@@ -0,0 +1,94 @@
+import { AxiosInstance } from 'axios';
+import { Except, SetOptional } from 'type-fest';
```

Comment:
I'm just curious, what is the difference between `setOptional` and the builtin `Partial`?
Same for `Except`  and `Exclude`

Reply:
Nevermind, looked at the docs 👍

Reply:
The `Omit` type does not restrict the omitted keys to be keys present on the given type, while `Except` does.

`Partial` marks all properties optional while `SetOptional` only those that mentioned keeping other untouched.